initializers/custom_links.yml

```diff
@@ -0,0 +1,21 @@
+## Possible Choices:
+##   new_window:
+##   - True
+##   - False
+##   content_type_id:
+##   - device
+##   - site
+##   - any-other-content-type
```

Comment:
Where can we find valid values for _any-other-content-type_?

Reply:
I believe running python manage.py nbshell and then entering "lsmodels()" will provide a list of models available to users. I believe content type is a convenient way of abstracting the type of object we are working with. It could also probably be called "model type" but the content type is what the core netbox project uses and built into Django so that is what I used.